minard/channeldb.py

```diff
@@ -303,6 +303,62 @@ def get_discriminator_threshold(crate, slot, channel):
 
     return threshold
 
+def get_all_thresholds(run):
+
+    conn = engine.connect()
+
+    zero = [[0 for i in range(16)] for j in range(19)]
+    thr = [[0 for i in range(16)] for j in range(19)]
+    disc = [9999]*9728
+
+    # Select most recent zdisc with ecalid field
+    result = conn.execute("select crate, slot, zero_disc from zdisc where "
```

Comment:
Should capitalize all sql commands (it's not necessary, but makes parsing the query a little easier)